### PR TITLE
Specify the desired SDK Version and Application.

### PR DESCRIPTION
**Pull request by matsudamper**
# Description
- Specify the desired SDK Version and Application.
- Add test

## Usage
https://github.com/kotest/kotest-extensions-robolectric/blob/234e4d4f74e93662f1b5d7cfe6a4c3696561eefd/test/src/test/kotlin/iokotest/kotest/extensions/robolectric/ContainedRobolectricRunnerTest.kt#L57-L66

# I wanted to, but I couldn't.
- Specifying multiple SDK Versions(This PR uses extend)
  - Replacing the ClassLoader was difficult for me.

# ideal
- `fun <T : Spec> instantiate(clazz: KClass<T>): Spec?` returns `List<Spec>` .

# Confirmed
- gradlew check
- gradlew extensions-robolectric:publishToMavenLocal

# Not Confirmed
- gradlew :extensions-robolectric:publish

### CODE REVIEW

**Comment by sksamuel** [comment]
@LeoColman are you able to review this ?

File: extensions-robolectric/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest package="iokotest.kotest">
```
Comment (LeoColman) [inline extensions-robolectric/src/main/AndroidManifest.xml]:
```suggestion
<manifest package="io.kotest">

```

**Review by LeoColman — APPROVED** [review]

**Comment by LeoColman** [comment]
Merged!
Thanks, @matsudamper!

File: extensions-robolectric/src/main/AndroidManifest.xml (hunk 1 above)
Comment (matsudamper) [reply extensions-robolectric/src/main/AndroidManifest.xml]: I thought that the tests for this extension do not work with 'io.kotest'.So I changed the package appropriately.
https://github.com/kotest/kotest-extensions-robolectric/blob/ed8ac0e158c950db1eb1a6710fa0edb4e43d2de6/extensions-robolectric/src/main/kotlin/io/kotest/extensions/robolectric/ContainedRobolectricRunner.kt#L33
I think I changed the main one by mistake at that time.

**Pull request merged**